### PR TITLE
Upgrade libraries for all projects 

### PR DESCRIPTION
**Pull request by tophersmith**
to remove latent, though unreachable, vulnerabilities in 3rd party code

### CODE REVIEW

File: plugin-bsimm/pom.xml [hunk 1]
```diff
@@ -10,7 +10,6 @@
 	<artifactId>plugin-bsimm</artifactId>
 
 	<properties>
-		<commons.io.version>2.6</commons.io.version>
 		<flyway.spring.test.version>7.0.0</flyway.spring.test.version>
```
Comment (madisoncool) [inline plugin-bsimm/pom.xml]: I think you can also delete this property

File: synapse-core/pom.xml [hunk 2]
```diff
@@ -10,10 +10,17 @@
 	<artifactId>synapse-core</artifactId>
 
 	<properties>
-		<unirest.version>3.3.00</unirest.version>
 		<flyway.spring.test.version>7.0.0</flyway.spring.test.version>
```
Comment (madisoncool) [inline synapse-core/pom.xml]: And this one

File: synapse-web/pom.xml [hunk 3]
```diff
@@ -22,12 +22,12 @@
 			<artifactId>synapse-core</artifactId>
 			<version>${project.version}</version>
 		</dependency>
-		<dependency>
```
Comment (madisoncool) [inline synapse-web/pom.xml]: Not sure if you meant to commit this file

**Review by madisoncool — CHANGES_REQUESTED** [review]
Just a couple things to delete

**Comment by tophersmith** [comment]
Good call. fixed @madisoncool

**Review by madisoncool — APPROVED** [review]

**Pull request merged**